packages/google_sign_in/google_sign_in/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-## 4.3.0
+## 4.3.1
```

Comment:
To keep it constant, we used to bump minor version for embedding migrations.  So let's make it 4.4.0